### PR TITLE
ref(ui): Extract global/styles

### PR DESCRIPTION
**Pull request by evanpurkhiser**
Should be a bit more manageable if we add more things here

### CODE REVIEW

File: src/sentry/static/sentry/app/styles/global.tsx [hunk 1]
```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {Global, css} from '@emotion/core';
+
+import {Theme} from 'app/utils/theme';
+
+const styles = (theme: Theme) => css`
```
Comment (priscilawebdev) [inline src/sentry/static/sentry/app/styles/global.tsx:6]: I would prefer to use the `styled(Global)` approach, but I believe it doesn't work atm, right?

**Review by priscilawebdev — APPROVED** [review]
This is great 👏

**Review by matejminar — APPROVED** [review]
nice

**Review by markstory — APPROVED** [review]

**Review by billyvg — APPROVED** [review]

File: src/sentry/static/sentry/app/styles/global.tsx (hunk 1 above)
Comment (EvanPurkhiser) [reply src/sentry/static/sentry/app/styles/global.tsx:6]: Is that a thing we're supposed to be able to do?

**Pull request merged**